src/main/java/gregtech/common/gui/widget/craftingstation/MemorizedRecipeWidget.java

```diff
@@ -75,4 +75,9 @@ public ItemStack slotClick(int dragType, ClickType clickTypeIn, EntityPlayer pla
         }
         return ItemStack.EMPTY;
     }
+
+    @Override
+    public boolean mouseClicked(int mouseX, int mouseY, int button) {
+        return super.mouseClicked(mouseX, mouseY, button);
+    }
```

Comment:
Is there a need for this override if you just return `super`?

Reply:
Oops, I likely just left this in for breakpoints.